test/functional/test-sanitizer.js

```diff
@@ -163,6 +165,51 @@ describe('sanitizeHtml', () => {
     expect(sanitizeHtml('<div class="c" src="d">b</div>')).to.be
         .equal('<div class="c" src="">b</div>');
   });
+
+  it('should output [text] and [class] attributes', () => {
+    expect(sanitizeHtml('<p [text]="foo" [class]="bar"></p>')).to.be
+        .equal('<p [text]="foo" [class]="bar"></p>');
+  });
+
+  it('should NOT output blacklisted values for class attributes', () => {
+    expect(sanitizeHtml('<p class="i-amphtml-">hello</p>')).to.be
+        .equal('<p>hello</p>');
+    expect(sanitizeHtml('<p class="i-amphtml-class">hello</p>')).to.be
+        .equal('<p>hello</p>');
+    expect(sanitizeHtml('<p class="foo-i-amphtml-bar">hello</p>')).to.be
+        .equal('<p>hello</p>');
+    expect(sanitizeHtml('<p [class]="i-amphtml-">hello</p>')).to.be
+        .equal('<p>hello</p>');
+    expect(sanitizeHtml('<p [class]="i-amphtml-class">hello</p>')).to.be
+        .equal('<p>hello</p>');
+    expect(sanitizeHtml('<p [class]="foo-i-amphtml-bar">hello</p>')).to.be
+        .equal('<p>hello</p>');
+  });
+
+  it('should NOT output security-sensitive amp-bind attributes', () => {
```

Comment:
On second thought, maybe this test should live in [the extension validator](./extensions/amp-mustache/0.1/test/validator-amp-mustache.html) -- feel free to do this in a follow-up.